parts/k8s/containeraddons/kubernetesmasteraddons-omsagent-daemonset.yaml

```diff
@@ -555,9 +551,6 @@ apiVersion: v1
 metadata:
   name: healthmodel-replicaset-service
   namespace: kube-system
-  labels:
```

Comment:
We should keep the reconcile labels so that addon manager can reconcile as needed.